### ISSUE TITLE
Consider using inclusive language in docs

### ISSUE THREAD

**Issue by KyleMit** [microsoft/vscode#88089]
From [Words To Avoid in Educational Writing](https://css-tricks.com/words-avoid-educational-writing/), the following words can be difficult for folks who are just getting started and have a steep learning curve to implementing suggestions which may seem trivial to advanced users:

* Simply
* Just
* Obviously
* Basically
* Of course
* Clearly
* Easy

For example, here's some language choices in the [September 2018 Release Notes](https://code.visualstudio.com/updates/v1_28#_project-level-snippets):

> * ***Simply*** use a string array as the `prefix` property.
> * Returning the value `undefined` ***simply*** prevents the debug session from starting
> * ***Simply*** use the **Preferences: Configure User Snippets** command or create `*.code-snippets` file
> * Search for keybindings ***just*** by typing keystrokes
> * Now can you rename the referenced file ***just*** by renaming the import:
> * You can accept that current suggestion ***just*** by typing `;`

Very little is lost by removing those words altogether:

> * <del>Simply</del> Use a string array as the `prefix` property.
> * Returning the value `undefined` <del>simply</del> prevents the debug session from starting
> * <del>Simply</del> Use the **Preferences: Configure User Snippets** command or create `*.code-snippets` file
> * Search for keybindings <del>just</del> by typing keystrokes
> * Now can you rename the referenced file <del>just</del> by renaming the import:
> * You can accept that current suggestion <del>just</del> by typing `;`

Setting up language linting or including this in editorial reviews is definitely non-trivial, but just a suggestion to focus on inclusive and welcoming language.

**Comment by David-Else** [comment]
I agree it is good to be welcoming and inclusive, but I followed the link, and the guy who wrote the article has zero qualifications in the subject and no data to backup anything he says. Why would anyone spend anytime implementing these ideas when there is no compelling evidence they have any benefit whatsoever?

A much better source to improve technical writing is `William Zinsser - On Writing Well`. He explains in the first part how to remove clutter and demonstrates how most writing can be cut in half and present the meaning in a far more concise and understandable manner.

Unfortunately someone has to go and study the book and spend the time learning how to write well. The idea of 'language linting' is just more `santagate` 1984 nonsense. If you want good writing then hire a good writer, don't start banning words for arbitrary reasons in case they upset/demotivate people.

**Comment by eine** [comment]
I agree with @David-Else, since I believe that words such as "simple", "obviously", "basically", "of course", etc. do add meaning to the sentence. It does make a difference for me in the sentences that the author of https://css-tricks.com/words-avoid-educational-writing/ used to exemplify a "non-suggested" usage.

> the following words can be difficult for folks who are just getting started and have a steep learning curve to implementing suggestions which may seem trivial to advanced users

I would very humbly and honestly suggest those folks not to try running faster than their feet can endure. Should they have issues with reality letting them know that they are new in some area and that they need to learn (https://en.wikipedia.org/wiki/Dunning%E2%80%93Kruger_effect), it is very little helpful to be overpaternalistic.

That said, I'm all in with using inclusive, welcoming and supportive language. Where "supporting" means adding references for new users to understand why things are simple, obvious or basic; instead of hiding that information from them by removing adverbs/adjectives.

**Comment by KyleMit** [comment]
> @David-Else 
> the guy who wrote the article has zero qualifications in the subject 

"the guy" who wrote the article is Chris Coyier who started CSS-Tricks and has written over 3,000+ technical blog posts over the last 10 years, co-founded codepen, has done a podcast with 300+ episodes, has published books on web development, spoken at dozens of national conferences.  If it comes down to the three of us, I'm going to trust him on reaching an audience on technical writing.

> @David-Else 
> is just more santagate 1984 nonsense

What the what! How did you escalate this in your head to there.  The idea is to be careful with word choice, a very common editorial practice, not ban words outright.  I've never even hinted at banning words; I said they *can* be hard for new users.

> @David-Else 
> He explains in the first part how to remove clutter and demonstrates how most writing can be cut in half and present the meaning in a far more concise and understandable manner.

Yes, I totally agree. And even if you didn't find this language particularly unwelcoming, it's removal still accomplishes your desired goal of writing in a more "concise and understandable manner"

Oh well, it's not a hill I need to die on; just a suggestion.

**Comment by eine** [comment]
> > the guy who wrote the article has zero qualifications in the subject
> 
> "the guy" who wrote the article is Chris Coyier who started CSS-Tricks and has written over...

We should be careful, as this looks like an *ad hominem* replied with an *ad verecundiam*...

> If it comes down to the three of us, I'm going to trust him on reaching an audience on technical writing.

I believe we'd better think about how to make release notes of VSCode more *friendly* for new users; and propose/discuss enhancements to the guidelines of the project. Trust seems out of context. Naturally, Chris is so welcome to come by and contribute to the discussion, given his experience and expectedly interesting point of view.

Please, don't take me wrong: I am so interested on the outcome of this, since I am a new in the field of Electron/TS. This is to say that I am one of those users that would benefit from more inclusive notes. Nonetheless, I don't *feel bad* for it; just wish I had more time to learn and understand.

---

Let's be specific:

> - Returning the value undefined <strike>simply</strike> prevents the debug session from starting

Without "simply", I would read it and ask myself: what else does it (not) prevent/trigger? is there any other prevent/trigger at all? Hence, I'd expect some link to an issue or doc page where the "full" (as in "not simple") procedure is explained.

With "simply", I understand that it prevents the debug session and it has no other meaningful (non-)effect. This is reassuring for me: someone with better knowledge is explicitly telling me not to overthink.

**Comment by David-Else** [comment]
@KyleMit 

> What the what! How did you escalate this in your head to there....  I've never even hinted at banning words;

You mentioned `Setting up language linting`, that is what I was referring to. To me that implies automated flagging of words for either deletion or modification.

Sorry, I didn't mean to come over so harsh. I am just very wary of people suggesting things like `language linting` after what has been happening lately in the open source world. In fact I will turn this positive, I think I will write a dystopian novel in which Google develops a real time 'language linter' so it can change what you say over the phone in real time. Eventually it can replace you with a sanitized automaton version of yourself :)
